seeq/addons/correlation/__main__.py

```diff
@@ -199,10 +199,16 @@ def cli_interface():
     user = args.username
     logging_attempts(user)
     seeq_url = args.seeq_url
+
+    try:
```

Comment:
See my comments in seeq/addons/correlation/_seeq_worksheet_writer.py 

Additionally, have you checked that `spy.client.session.public_url` and `spy.client.session.private_url` work from the terminal of the SDL project?

Reply:
yes, checked in R52 and above both the functions and them working in the app as a whole on test data (built wheel file, pip install then run developer version on projects). I am guessing the 'client' in your comment is a type?